instrumentation/opentelemetry-instrumentation-starlette/test-requirements.latest.txt

```diff
@@ -0,0 +1,112 @@
+# This file was autogenerated by uv via the following command:
+#    uv pip compile --python 3.8 --universal --extra instruments instrumentation/opentelemetry-instrumentation-starlette/pyproject.toml -c dev-requirements.txt instrumentation/opentelemetry-instrumentation-starlette/test-requirements.in -o instrumentation/opentelemetry-instrumentation-starlette/test-requirements.latest.txt
+-e file:///usr/local/google/home/aaronabbott/repo/opentelemetry-python-contrib/instrumentation/opentelemetry-instrumentation-asgi
```

Comment:
Thanks for sharing :)

Reply:
I still don't know why UV does this but I found an incantation that removes it.